### ISSUE TITLE
[DOC] Advanced => Addons => PCI Devices, a sublink is broken

### ISSUE THREAD

**Issue by albinsun** [harvester/harvester#6890]
**Is your doc request related to a problem? Please describe or add the related issue ID.**

https://docs.harvesterhci.io/v1.3/advanced/addons/pcidevices/#installing-drivers-for-your-pci-device-inside-the-vm
![image](https://github.com/user-attachments/assets/44f4c427-8aa5-4ca3-83ef-3887e03027ef)

**Describe the solution you'd like**
n/a

## Additional context
Broken since `v1.1`.

**Comment by albinsun** [comment]
Hyperlink still broken
* `v1.3`
  ![Image](https://github.com/user-attachments/assets/f88fb665-2e69-4fd6-a59b-a32f0e595062)

* `v1.4`
  ![Image](https://github.com/user-attachments/assets/f5dae7ea-c788-42c7-a69e-6e5fe5f00ad6)

* `v1.5` 
  ![Image](https://github.com/user-attachments/assets/b7ed5a41-7f41-47a5-9ef7-7413c307bc27)

**Comment by jillian-maroket** [comment]
@albinsun Please check the [blog post](https://tobilehman.com/archive/tobilehman.com/posts/suse-harvester-pci.html) and let me know if we should still link to it. The [image link](https://tobilehman.com/archive/tobilehman.com/posts/suse-harvester-pci/img/vfio-device-assignment-viommu.png) has changed.

**Comment by albinsun** [comment]
> [@albinsun](https://github.com/albinsun) Please check the [blog post](https://tobilehman.com/archive/tobilehman.com/posts/suse-harvester-pci.html) and let me know if we should still link to it. The [image link](https://tobilehman.com/archive/tobilehman.com/posts/suse-harvester-pci/img/vfio-device-assignment-viommu.png) has changed.

No, imo we shouldn't link to it anymore cuz the image is about vfio-device-assignment but not NVIDIA drivers being installed in a VM.

**Comment by jillian-maroket** [comment]
All affected doc versions have been updated.
